WordPress/Classes/ViewRelated/Menus/Views/MenusHeaderView.h

```diff
@@ -0,0 +1,62 @@
+#import <UIKit/UIKit.h>
+
+@class Blog;
+@class MenuLocation;
+@class Menu;
+
+@protocol MenusHeaderViewDelegate;
+
+@interface MenusHeaderView : UIView
```

Comment:
Since you took the work of documenting what this is for the issue description, may I suggest copy-pasting that description right here?